tutorials/video/getting-started/video_split_clip_example.py

```diff
@@ -215,9 +219,16 @@ def create_video_splitting_pipeline(args: argparse.Namespace) -> Pipeline:  # no
                 )
             )
 
+        # Determine model directory
+        model_dir = (
+            args.captioning_model_path
```

Comment:
I would suggest name this as `nemotronh_vl_model_path`. Once we have the nemotronh uploaded to HF, we can simplify the logic here